api/src/main/java/io/kroxylicious/testing/kafka/api/KafkaCluster.java

```diff
@@ -29,7 +29,7 @@ public interface KafkaCluster extends AutoCloseable {
     void close() throws Exception;
 
     /**
-     * @return the size of the cluster
+     *Returns the size of the cluster.
```

Comment:
I have to say I think lift is just wrong here. 

We should keep the `@return` doc as well.

Reply:
Agree, changed by sonatype